### ISSUE TITLE
Does not work with angular9 anymore

### ISSUE THREAD

**Issue by SebastianStehle** [dmacfarlane/angular-mentions#141]
Hi,
in my case it stopped working with angular9. The list is popping up but without any text. I think the reason is that you assign the default template  ngOnInit instead ngAfterViewInit.
 
I got it working with a custom template.

**Comment by jcg3** [comment]
I ran into the same issue, thanks for being first Sebastian and pointing me in the right direction!

In case anyone else needs help, here's the code to work around the issue in your component template. 

```
<textarea [mention]="mention_list" 
          [mentionListTemplate]="fixMentionDisplay"
          [(ngModel)]="value"></textarea>

<ng-template #fixMentionDisplay let-item="item">
  {{item['label']}}
</ng-template>

```

The variable mention_list is an array of strings: mention_list = ['Alpha', 'Bravo', 'Charlie'];

**Comment by dmacfarlane** [comment]
This issue was also reported in #139 and it has been fixed in the 1.1.4 release.

**Comment by jcg3** [comment]
Awesome, thanks!

**Comment by SebastianStehle** [comment]
I will test it and close the issue after that.

**Comment by SebastianStehle** [comment]
Sorry, it works.

**Comment by naimmalek** [comment]
@dmacfarlane I've got a workaround. Can I submit a PR for this ?

**Comment by dmacfarlane** [comment]
Sure, but please keep the CONTRIBUTING.md notes and #144 in mind.

**Comment by dmacfarlane** [comment]
Reopening as the fix for angular 9 was undone in 1.1.5 release due to #144.

**Comment by naimmalek** [comment]
@dmacfarlane I have created PR

**Comment by naimmalek** [comment]
@dmacfarlane I've created PR resolving this issue: #144

**Comment by dmacfarlane** [comment]
Thank you @naimmalek! The fix is working for angular 8 and 9 in the 1.1.6 release.
